src/cvc.mli

```diff
@@ -0,0 +1,33 @@
+module RSA :
+sig
+  module Public :
+  sig
+    type t =
+      { n: Z.t
+      ; e: Z.t
+      }
+    [@@deriving ord,yojson,eq,show]
+  end
+end
+
+module ECDSA :
+sig
+  module Public :
+  sig
+    type t =
+      { modulus : Z.t
+      ; coefficient_a : Z.t
+      ; coefficient_b : Z.t
+      ; base_point_g : Z.t
+      ; base_point_r_order : Z.t
+      ; public_point_y : Z.t
+      ; cofactor_f : Z.t
+      }
+    [@@deriving ord,yojson,eq,show]
+  end
+end
+
+type t =
```

Comment:
Yeah I meant `RSA.Public.decode` indeed. Let's keep the `Public` part. Please make it so that the public API is:
- `Cvc.RSA.Public.decode: Cstruct.t -> (Cvc.RSA.Public.t, string) Result.result`
- `Cvc.ECDSA.Public.decode: Cstruct.t -> (Cvc.ECDSA.Public.t, string) Result.result`
- the existing `Cvc.t` and `Cvc.decode` are *not* exposed.